### PR TITLE
Move crypto functions to crypto package

### PR DESCRIPTION
**Pull request by IronGauntlets**
## Description

Move crypto functions to crypto package.

## Changes:

- Move cpp files to `crypto/cpp` package
- Move pedersen files to `crypto` package

## Types of changes

- Refactoring (no functional changes, no api changes)

## Testing

**Requires testing**: No

## Documentation

**If this requires a documentation update, did you add one?** No

### CODE REVIEW

File: core/crypto/cpp/crypto_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package crypto
+package cpp
```
Comment (omerfirmak) [inline core/crypto/cpp/crypto_test.go:1]: what about `starkware` ?

Comment (IronGauntlets) [reply core/crypto/cpp/crypto_test.go:1]: Yeah sure i will update

Comment (IronGauntlets) [reply core/crypto/cpp/crypto_test.go]: Done

**Comment by jelilat** [comment]
The folder name change here is `starkware` while [this](https://github.com/NethermindEth/juno/pull/443) is `cpp`. Which are we using?

**Review by jelilat — COMMENTED** [review]
The folder name change here is `starkware` while [this](https://github.com/NethermindEth/juno/pull/443 ) is `cpp`. Which are we using?

**Comment by omerfirmak** [comment]
> The folder name change here is `starkware` while [this](https://github.com/NethermindEth/juno/pull/443) is `cpp`. Which are we using?

@IronGauntlets will probably rebase that PR branch after this is merged

**Review by omerfirmak — APPROVED** [review]
Merge at will

**Pull request merged**